api_testrail.py

```diff
@@ -318,6 +345,34 @@ def testrail_users(self):
         df = pd.DataFrame(user_data)
         self.db.report_testrail_users_insert(df)
 
+    def testrail_runs(self, project, num_days):
+        start_date = dt.start_date(num_days)
+        self.db.projects_insert(17, 'desktop', 'firefox-desktop')
```

Comment:
Not sure why you insert it.. the desktop project is already in the `projects` table (staging and production):
<img width="597" height="238" alt="imagen" src="https://github.com/user-attachments/assets/4587bf4b-f436-4d44-b375-5657a7af9f3b" />